api_tests/tests/dry/rfc003_schema.ts

```diff
@@ -0,0 +1,220 @@
+/**
```

Comment:
Wow, this is going to be hard to review. Did you `git mv` the files?

Reply:
I guess we have to trust that the actual tests didn't change.

Reply:
Sorry for that. The tests themselves really did not change. All I did was consolidate them and move into new directory/files.

Reply:
Yes I only noticed further along :)